app/concierge/mycore/migrations/0003_auto_20200120_1854.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.0.2 on 2020-01-20 18:54
+
+from django.db import migrations
+
+OBJECTS_TO_CREATE = 5
+
+
+def combine_names(apps, schema_editor):
+    Tenant = apps.get_model('mycore', 'Tenant')
+    Room = apps.get_model('mycore', 'Room')
+    for i in range(OBJECTS_TO_CREATE):
+        room = Room(number=i)
```

Comment:
room with number 0 will be created, I think a room has to start from the first number

Reply:
Fixed! I have added validator MinvalueValidator to that field with value 1